getaddressbalance.html

```diff
@@ -36,6 +36,21 @@ <h1>getaddressbalance - Zcash 6.2.0 RPC</h1>
 
 Returns the balance for addresses.
 
+WARNING: getaddressbalance is disabled.
```

Comment:
Where did this change come from? I don't see anything related to `getaddressbalance` in this release; this appears to have been disabled forever.